.evergreen/generate_evergreen_tasks.js

```diff
@@ -663,6 +663,13 @@ BUILD_VARIANTS.push({
   tasks: ['serverless_task_group']
 });
 
+BUILD_VARIANTS.push({
+  name: 'rhel8-test-gcp-kms',
+  display_name: 'GCP KMS Test',
```

Comment:
```suggestion
  display_name: 'FLE KMS Refresh Tests',
```


we will soon have both AWS and Azure as well - maybe we can just have a generic task group?

Reply:
I don't think you'll be able to reuse this variant, it's specific to GCP / GCP setup scripts